### PR TITLE
Added DBMS Course

### PR DESCRIPTION
**Pull request by metabiswadeep**
## What does this PR do?
Added a DBMS course

### Description
Added a DBMS course for gaining access to free resources to learn DBMS

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
why not in 'databases'?
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

**Comment by metabiswadeep** [comment]
It is because a new category will help contributors to find out new resources in that category as well, which will help in improvement of resources in that category as well. @eshellman

**Comment by metabiswadeep** [comment]
If it is not preferred, please inform and I will add the resource to Databases category

**Comment by metabiswadeep** [comment]
Done @eshellman

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -336,6 +336,7 @@
 
 * [Database Systems](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-830-database-systems-fall-2010/) (MIT's opencourseware)
 * [Database Systems - Cornell University Course (SQL, NoSQL, Large-Scale Data Analysis)](https://www.youtube.com/watch?v=4cWkVbC2bNE) - Professor Immanuel Trummer, freeCodeCamp.org (YouTube)
+* [DBMS Placement Series](https://youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - LoveBabbar
```
Comment (eshellman) [inline courses/free-courses-en.md:339]:
```suggestion
* [DBMS Placement Series](https://youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp by Babbar
```

**Comment by metabiswadeep** [comment]
Done @eshellman

**Review by davorpa — APPROVED** [review]

**Pull request merged**